templates/data-source.md.tmpl

```diff
@@ -55,6 +55,7 @@
     {{ else if eq .Name "mongodbatlas_third_party_integration" }}
     {{ else if eq .Name "mongodbatlas_x509_authentication_database_user" }}
     {{ else if eq .Name "mongodbatlas_privatelink_endpoint_service_data_federation_online_archive" }}
+    {{ else if eq .Name "mongodbatlas_stream_privatelink_endpoint" }}
```

Comment:
Are doc updates not done in a different PR?

Reply:
yes it will be done in a different PR, but since I am adding the DS into the provider, without this, the generate-doc-all GH action would fail